lib/ansible/module_utils/ovirt.py

```diff
@@ -281,7 +281,7 @@ def search_by_name(service, name, **kwargs):
     # Check if 'list' method support search(look for search parameter):
     if 'search' in inspect.getargspec(service.list)[0]:
         res = service.list(
-            search="name={name}".format(name=name)
+            search='name="{name}"'.format(name=name)
```

Comment:
Please add comment: There must be double quotes around name, because some oVirt resources it's possible to create then with space in name.